lib/src/views/settings_view.dart

```diff
@@ -9,6 +9,11 @@ class SettingsView extends StatefulWidget {
 class _SettingsViewState extends State<SettingsView> {
   String _adapterValue;
 
+  String _hipChatTokenValue;
```

Comment:
should these be updated to be slack tokens?

Reply:
We didn't implement Slack yet, so for now it's just hipChatToken. Adding Slack token will be next thing  that has to be done.